.trivyignore

```diff
@@ -0,0 +1,8 @@
+# ignore in old airflow versions; it's fixed ONLY in new airflow version
+# lodash: Prototype pollution in utilities function fixed in >=4.17.11
+CVE-2018-16487
```

Comment:
Hmmmm, not sure about a global ignore here -- if it somehow comes back in 1.10.7 we'd want to know about it.

That probably means we need more than one of these files (a shared one, and one per airflow version (or per image?)). Hmmm